### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by adam-weiler**
I tried to update your Readme file to point towards metricly.com instead of netuitive.com.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,11 +3,11 @@ Netuitive StatsD Server
...
+The Netuitive StatsD Server interprets, aggregates, and forwards custom metrics generated from your application to [Netuitive](https://www.metricly.com). Netuitive StatsD is compatible with the following formats: StatsD, DogStatsD Metrics, DogStatsD Events, and DogStatsD Service Checks. Using the values instrumented from your application's key actions and data (method calls, database queries, etc.), Netuitive aggregates the values, associates them with corresponding metrics, and analyzes them in our analytics cycles.
 
-For more information on the Netuitive StatsD Server, see our [help docs](https://help.netuitive.com/Content/Misc/Datasources/Netuitive/new_netuitive_datasource.htm#kanchor275), or contact Netuitive support at [support@netuitive.com](mailto:support@netuitive.com).
+For more information on the Netuitive StatsD Server, see our [help docs](https://www.metricly.com/support/integrations/linux/#kanchor275), or contact Netuitive support at [sales@metricly.com](mailto:sales@metricly.com).
```
Comment (TheConnMan) [inline README.md:10]: We'd still like support questions to go to support.

```suggestion
For more information on the Netuitive StatsD Server, see our [help docs](https://www.metricly.com/support/integrations/linux/#kanchor275), or contact Metricly support at [support@metricly.com](mailto:support@metricly.com).
````

File: README.md [hunk 2]
```diff
@@ -3,11 +3,11 @@ Netuitive StatsD Server
 
 [![Build Status](https://travis-ci.org/Netuitive/netuitive-statsd.svg?branch=master)](https://travis-ci.org/Netuitive/netuitive-statsd) [![Coverage Status](https://coveralls.io/repos/github/Netuitive/netuitive-statsd/badge.svg?branch=master)](https://coveralls.io/github/Netuitive/netuitive-statsd?branch=master)
 
->**Note:** This is already installed as part of the [Netuitive Linux Agent](https://help.netuitive.com/Content/Misc/Datasources/Netuitive/new_netuitive_datasource.htm) package and does not need to be installed separately.
+>**Note:** This is already installed as part of the [Netuitive Linux Agent](https://www.metricly.com/support/integrations/linux/) package and does not need to be installed separately.
```
Comment (TheConnMan) [inline README.md:6]: Just a change to Metricly.

```suggestion
>**Note:** This is already installed as part of the [Metricly Linux Agent](https://www.metricly.com/support/integrations/linux/) package and does not need to be installed separately.
```

File: README.md [hunk 3]
```diff
@@ -3,11 +3,11 @@ Netuitive StatsD Server
...
 
-The Netuitive StatsD Server interprets, aggregates, and forwards custom metrics generated from your application to [Netuitive](https://www.netuitive.com). Netuitive StatsD is compatible with the following formats: StatsD, DogStatsD Metrics, DogStatsD Events, and DogStatsD Service Checks. Using the values instrumented from your application's key actions and data (method calls, database queries, etc.), Netuitive aggregates the values, associates them with corresponding metrics, and analyzes them in our analytics cycles.
+The Netuitive StatsD Server interprets, aggregates, and forwards custom metrics generated from your application to [Netuitive](https://www.metricly.com). Netuitive StatsD is compatible with the following formats: StatsD, DogStatsD Metrics, DogStatsD Events, and DogStatsD Service Checks. Using the values instrumented from your application's key actions and data (method calls, database queries, etc.), Netuitive aggregates the values, associates them with corresponding metrics, and analyzes them in our analytics cycles.
```
Comment (TheConnMan) [inline README.md:8]: Another Metricly name change.

```suggestion
The Netuitive StatsD Server interprets, aggregates, and forwards custom metrics generated from your application to [Metricly](https://www.metricly.com). Netuitive StatsD is compatible with the following formats: StatsD, DogStatsD Metrics, DogStatsD Events, and DogStatsD Service Checks. Using the values instrumented from your application's key actions and data (method calls, database queries, etc.), Netuitive aggregates the values, associates them with corresponding metrics, and analyzes them in our analytics cycles.
```

**Review by TheConnMan — CHANGES_REQUESTED** [review]
Thanks for the contribution! Just a couple suggested Metricly name changes.

**Comment by Grumblesmith** [comment]
Please rebase your branch against develop

**Comment by rclagett** [comment]
A new docs site has launched!

**Pull request closed without merging**